capslock/capslock.js

```diff
@@ -0,0 +1,13 @@
+// https://codeforces.com/problemset/problem/131/A
+
+var string = "cAPS";
+
+function firstBigRestSmall(string) {
```

Comment:
mozna nazwac ciut inaczej ten parametr bo String to typ danych. I sa pewne funkcje ktore sa wywolywane na String bezposrednio